geode-core/src/test/java/org/apache/geode/internal/cache/map/RegionMapDestroyTest.java

```diff
@@ -223,6 +241,23 @@ private void givenOriginIsRemote() {
     event.setOriginRemote(true);
   }
 
+  @Test
+  public void destroWithDuplicateVersionInvokesListener() {
```

Comment:
typo "destro"

Reply:
Wasn't "destro" a GI Joe villain in the 80s?